Berksfile

```diff
@@ -0,0 +1,18 @@
+source 'https://supermarket.chef.io'
+
+cookbook 'base',
+         git: 'git@github.com:osuosl-cookbooks/base'
+
```

Comment:
This is the `rubocop` corrected way to align these lines, and I figured instead of having one of the git lines be too long, I'd make them all follow the same style.